images/nginx/rootfs/build.sh

```diff
@@ -151,6 +151,18 @@ get_src()
   tar xzf "$f"
   rm -rf "$f"
 }
+OPENSSLEXTRAPKG=""
+if [[ -z ${ARCH} || ${ARCH} == "x86_64" || ${ARCH} == "aarch64" ]]; then
+        wget 'http://openresty.org/package/admin@openresty.com-5ea678a6.rsa.pub'
+        mv 'admin@openresty.com-5ea678a6.rsa.pub' /etc/apk/keys/
+        . /etc/os-release
+        MAJOR_VER=`echo $VERSION_ID | sed 's/\.[0-9]\+$//'`
```

Comment:
Great! I like sed!

Reply:
Glad you liked it, I took from here: http://openresty.org/en/linux-packages.html#alpine

:P